track/track-api-log/src/main/kotlin/TrackContext2Log.kt

```diff
@@ -38,7 +38,7 @@ private fun TrackError.toLog() = ErrorLogModel(
 )
 
 private fun TrackTicket.toLog() = TicketLog(
-    id = id.takeIf { it != TrackTicketId.NONE }?.asString(),
+    id = id.takeIf { it != TrackTicketId.NONE }?.asInt().toString(),
```

Comment:
```suggestion
    id = id.takeIf { it != TrackTicketId.NONE }?.asInt()?.toString() ?: "",
```
здесь у вас могут быть неожиданные вещи, если явно не пропишете

Reply:
Ясно, спасибо.